packages/headless/src/features/standalone-search-box-set/standalone-search-box-set-state.ts

```diff
@@ -0,0 +1,37 @@
+import {OmniboxSuggestionMetadata} from '../query-suggest/query-suggest-analytics-actions';
```

Comment:
Why not replace Omnibox by SearchBox here and elsewhere? It just feels like legacy perhaps?

Reply:
Yep, I was thinking it would be less confusing to keep the same name used by the `coveo.analytics` package.